pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
@dan144 @dbangera23 I see two failures when I run the tests myself (the other error may just be Travis being weird), both in testBatchExtractor, just with assertions that fail. Can you guys look into it and see what's going on, then get the tests to pass? Thanks in advance!